ir-engine/etherealengine-archive
# Add thumbnails to default project ( IR-1279 )

## Summary

Associated with https://tsu.atlassian.net/browse/IR-1279 . Adding thumbnails for default project to the repo and resource JSON, so that the thumbnail URLs and files exist in lock-step when initialized.

## Subtasks Checklist

## Breaking Changes

## References
closes IR-1279

## QA Steps

`npm run dev-reinit`
`npm run dev`
`https://localhost:3000/studio?project=default-project`
Click the `assets` folder in the Files tab
After a few seconds, all the assets in that folder should have a custom image or a blank image as their thumbnail.

This may be a little vague; if you're in QA and you want more clarity, please reach out 😃
